src/release/breaking-changes/3-10-deprecations.md

```diff
@@ -336,6 +341,161 @@ Relevant PRs:
 
 ---
 
+### AnimationSheetBuilder display & sheetSize
+
+Package: flutter_test
+Supported by Flutter Fix: yes
```

Comment:
This may be better phrased as "partially" supported instead of a direct yes. The fix does drive you to the `collate` method, but the end result of the fix needs further modifications by the user to compile. Not sure if that's worth the distinction, but still.

Reply:
That's ok, we have a lot of Flutter fixes that get you part of the way there. 😉